File: src/main/java/eladkay/quaritum/common/compat/mt/CraftTweaker.java
package eladkay.quaritum.common.compat.mt;

import minetweaker.MineTweakerAPI;
import minetweaker.api.item.IItemStack;
import minetweaker.api.minecraft.MineTweakerMC;
import net.minecraft.item.ItemStack;
import net.minecraftforge.fml.common.Loader;

public class CraftTweaker {
    public static void init() {
        if (!Loader.isModLoaded("MineTweaker3")) return;
        MineTweakerAPI.registerClass(DiagramMT.class);
    }

    public static ItemStack[] getStacks(IItemStack[] stacks) {
        ItemStack[] ret = new ItemStack[stacks.length];
        for (int i = 0; i < stacks.length; i++)
            ret[i] = MineTweakerMC.getItemStack(stacks[i]);
        return ret;
    }
}

File: src/main/java/eladkay/quaritum/common/compat/mt/CraftTweaker.kt
package eladkay.quaritum.common.compat.mt

import minetweaker.MineTweakerAPI
import minetweaker.api.item.IItemStack
import minetweaker.api.minecraft.MineTweakerMC
import net.minecraft.item.ItemStack
import net.minecraftforge.fml.common.Loader

object CraftTweaker {
    fun init() {
        if (!Loader.isModLoaded("MineTweaker3")) return
        MineTweakerAPI.registerClass(DiagramMT::class.java)
    }

    fun getStacks(stacks: Array<IItemStack>): Array<ItemStack> {
        val ret = arrayOfNulls<ItemStack>(stacks.size)
        for (i in stacks.indices)
            ret[i] = MineTweakerMC.getItemStack(stacks[i])
        return ret
    }
}
